src/helpers/pdfkitAddPlaceholder.js

```diff
@@ -29,28 +31,61 @@ const pdfkitAddPlaceholder = ({
         Contents: Buffer.from(String.fromCharCode(0).repeat(signatureLength)),
         Reason: new String(reason), // eslint-disable-line no-new-wrappers
         M: new Date(),
+        ContactInfo: new String('emailfromp1289@gmail.com'), // eslint-disable-line no-new-wrappers
+        Name: new String('Name from p12'), // eslint-disable-line no-new-wrappers
+        Location: new String('Location from p12'), // eslint-disable-line no-new-wrappers
     });
 
+    // Check if pdf already contains acroform field
+    const acroFormPosition = pdfBuffer.lastIndexOf('/Type /AcroForm');
+    const isAcroFormExists = acroFormPosition !== -1;
+    let fieldIds = [];
+    let acroFormId;
+
+    if (isAcroFormExists) {
+        const pdfSlice = pdfBuffer.slice(acroFormPosition - 12);
+        const acroForm = pdfSlice.slice(0, pdfSlice.indexOf('endobj')).toString();
+        const acroFormFirsRow = acroForm.split('\n')[0];
+        acroFormId = parseInt(acroFormFirsRow.split(' ')[0]);
+
+        const acroFormFields = acroForm.slice(acroForm.indexOf('/Fields [') + 9, acroForm.indexOf(']'));
+        fieldIds = acroFormFields
+            .split(' ')
+            .filter((element, index) => index % 3 === 0)
+            .map(fieldId => new PDFKitReferenceMock(fieldId));
+    }
+    const signatureName = 'Signature';
+
     // Generate signature annotation widget
     const widget = pdf.ref({
         Type: 'Annot',
         Subtype: 'Widget',
         FT: 'Sig',
-        Rect: [0, 0, 0, 0],
+        Rect: [188.67, 679.07, 384.78, 748.04],
```

Comment:
This should somehow be configurable or left as it used to be (as an invisible signature)